### ISSUE TITLE
Add CategoricalEncoderTransforms

### ISSUE THREAD

**Issue by iKintosh** [tinkoff-ai/etna#355]
### 🚀 Feature Request

Adapt categorical encoders from Sklearn into etna

### Motivation

Really frustrating when such simple transform is absent in etna

### Proposal

Create sklearn converter for
https://scikit-learn.org/stable/modules/generated/sklearn.preprocessing.LabelEncoder.html
https://scikit-learn.org/stable/modules/generated/sklearn.preprocessing.LabelBinarizer.html

Create 2 classes LabelEncoderTransform and LabelBinarizerTransform
Should work with int and str types.

If meet unknown label:
1. LabelEncoderTransform should use mean label value
2. LabelBinarizerTransform should use mean label vector value

The return column should be categorical.

Inverse transform is not necessary (but it should be easy to do)

### Checklist

- [x] I discussed this issue with ETNA Team
